### PR TITLE
Transmute packages for smaller installer

### PR DESCRIPTION
**Pull request by hmaarrfk**
Closes: https://github.com/conda-forge/miniforge/pull/55

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
PyPy builds fail with:

```
Traceback (most recent call last):
  File "/opt/conda/bin/constructor", line 8, in <module>
    sys.exit(main())
  File "/opt/conda/lib/python3.8/site-packages/constructor/main.py", line 243, in main
    main_build(dir_path, output_dir=out_dir, platform=args.platform,
  File "/opt/conda/lib/python3.8/site-packages/constructor/main.py", line 137, in main_build
    create(info, verbose=verbose)
  File "/opt/conda/lib/python3.8/site-packages/constructor/shar.py", line 126, in create
    t.add(join(info['_download_dir'], fn), 'pkgs/' + fn)
  File "/opt/conda/lib/python3.8/tarfile.py", line 1953, in add
    tarinfo = self.gettarinfo(name, arcname)
  File "/opt/conda/lib/python3.8/tarfile.py", line 1832, in gettarinfo
    statres = os.lstat(name)
FileNotFoundError: [Errno 2] No such file or directory: '/home/conda/.conda/constructor/linux-aarch64/python-3.6.12-5_73_pypy.conda'
```

File: Miniforge3/construct.yaml [hunk 1]
```diff
@@ -11,6 +11,7 @@ channels:
   # - https://conda.anaconda.org/conda-forge
   - conda-forge
 
+transmute_file_type: '.conda'
```
Comment (jakirkham) [inline Miniforge3/construct.yaml:14]:
```suggestion
{% if not name.endswith("pypy3") %}
transmute_file_type: '.conda'
{% endif %}
```

**Review by jakirkham — COMMENTED** [review]
Maybe we can just skip doing this on PyPy?

**Comment by hmaarrfk** [comment]
We could. But I would want to refer to a github issue opened at the correct place so we can give people a chance at fixing this.

**Comment by jakirkham** [comment]
Actually would suggest a new issue as the scope would have been narrowed, but yeah we would want to track the remaining issues somehow

**Comment by hmaarrfk** [comment]
unfortunately it isn't just pypy causing failures.

**Comment by hmaarrfk** [comment]
there seems to still be an issue with transmutting. Not willing to debug it myself.

**Pull request closed without merging**